packages/x-components/src/x-modules/empathize/components/empathize.vue

```diff
@@ -73,6 +76,23 @@
      */
     protected isOpen = false;
 
+    /**
+     * The default slot is empty to avoid opening empathize with no content.
+     *
+     * @internal
+     */
+    protected isEmptySlot: boolean | Element = true;
```

Comment:
I'd say that this variable will always be a boolean, and in that case you don't need to type it if in the declaration itself is assigned to `true`:
```suggestion
    protected isEmptySlot = true;
```